registration/backends/activation/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""
```

Comment:
why did you pull the directory in? it's in your virtual environment when you pip install the module:

`venv/lib/python3.8/site-packages/django_registration`